File: src/main/java/com/hy/frame/adapter/IAdapterListener.java
package com.hy.frame.adapter;

import android.view.View;

public interface IAdapterListener<T> {
    /**
     * 点击事件
     * 
     * @param v
     * @param item
     * @param position
     */
    void onViewClick(View v, T item, int position);
}

File: Frame/src/main/java/com/hy/frame/adapter/IAdapterListener.kt
package com.hy.frame.adapter

import android.view.View

/**
 * IAdapterListener Click
 * @author HeYan
 * @time 2017/5/23 9:57
 */
interface IAdapterListener<in T> {

    /**
     * 点击事件
     * @param v
     * @param item
     * @param position
     */
    fun onViewClick(v: View, item: T, position: Int)
}
